Source/monster.cpp

```diff
@@ -1086,7 +1086,7 @@ void MonsterAttackMonster(Monster &attacker, Monster &target, int hper, int mind
 	if (hit >= hper)
 		return;
 
-	int dam = (mind + GenerateRnd(maxd - mind + 1)) << 6;
+	int dam = RandomIntBetween(mind, maxd) << 6;

```

Comment:
```suggestion
	int dam = RandomIntBetween(mind << 6, maxd << 6);
```
to match MonsterAttackPlayer (as discussed in discord)?

Reply:
I don't think this would be in the scope of this PR. I think it's hard to prove or disprove if fractional damage or lack thereof is intentional in any given scenario based on how it was done originally. I'd rather approach this later.